src/amp-story-player/README.md

```diff
@@ -0,0 +1,449 @@
+- `backgroundImageUrl`: string with url or data string (escaped).
+  - Changes the icon image to the provided url or data string (for inline svgs).
+
+### Custom control
+
+You can add a custom control to the stories inside the player with a custom control. Simply specify a “name” and an “backgroundImageUrl”, and any optional properties:
+
+- `name` (**required**): a string with the name of the control. e.g. “lightbox”. **The dispatched event will depend on this name.** The custom event will be the name of the control prefixed with `amp-story-player-*`. E.g. `amp-story-player-lightbox`:
+
+```javascript
+const player = document.body.querySelector("amp-story-player");
+
+// Listen to when the specified control was clicked.
+player.addEventListener("amp-story-player-lightbox", () => {
+  // This will trigger when the control with the "lightbox" name is clicked.
+  performCustomAction();
+});
+```
+
+- `backgroundImageUrl` (**required**): Accepts URLs, as well as svgs and `data` paths (note that strings must be JSON escaped). See [example 3](#Example-#3---Changing-the-icon-of-the-close-button).
+  - Changes the control icon.
+- `position`: “start” or “end”.
+  - Places the icon either on the left or right on LTR languages.
+- `visibility`: “hidden” or “visible” (default).
+  - Toggles the control’s visibility. If omitted, the default is visible.
+
+### Example #1 - Close button on the start position
+
+Since by default the close button will be placed to the end, all we have to do is move the close button to the start.
+
+<table>
+  <tr>
+    <td>
+      <pre lang="html">
+<amp-story-player>
+ <script type="application/json">
+   {
+     "controls": [
+       {
+         "name": "close",
+         "position": "start"
+       }
+     ],
+   }
+ </script>
+ ...
+</amp-story-player>
+      </pre>
+    </td>
+    <td><img width="360" height="600" src="https://github.com/ampproject/amphtml/blob/master/src/amp-story-player/img/lightbox-close-button.png?raw=true"></td>
+  </tr>
+</table>
+
+### Example #2 - Showing skip-to-next story on desktop
+
+On desktop, you can now display a button that navigates from the current story to the next one. It will also automatically be disabled once the user reaches the end of the stories in the player.
+
+<table>
+  <tr>
+    <td>
+      <pre lang="html">
+<amp-story-player>
+ <script type="application/json">
+   {
+     "controls": [
+       {
+         "name": "skip-next"
+       }
+     ],
+   }
+ </script>
+ ...
+</amp-story-player>
+      </pre>
+    </td>
+    <td><img src="https://github.com/ampproject/amphtml/blob/master/src/amp-story-player/img/skip-next-desktop.png?raw=true"></td>
+  </tr>
+</table>
+
+### Example #3 - Changing the icon of the close button
+
+<table>
+  <tr>
+    <td>
+      <pre lang="html">
+<amp-story-player>
+ <script type="application/json">
+   {
+     "controls": [
+       {
+         "name": "close",
+         "backgroundImageUrl": "https://example.com/assets/red-x.png",
```

Comment:
Can we have the icon inlined instead? If the image has to be loaded it might not be visible right away on the story and suddenly appear later. It's cool that this is possible but we should push people towards inlining their icons here.

Reply:
Sure, done.